docs/general/ad-filtering/create-own-filters.md

```diff
@@ -3122,7 +3136,36 @@ $$div[tag-content="banner"][min-length="400"]
 
 This rule will remove all the `div` elements, whose code contains the substring `banner` and the length of which exceeds `400` characters.
 
-**Exceptions**
+### Pseudo-classes
+
+#### `:contains()`
+
+##### Syntax
+```
+:contains(unquoted text)
+```
+or
+```
+:contains(/reg(ular )?ex(pression)?/)
+```
+
+:::note Compatibility
+
+`:-abp-contains()` and `:has-text()` are synonyms for `:contains()`.
```

Comment:
why does this string have a Compatibility admonition?

Reply:
It's about compatibility with other ad blockers' syntax. What is the correct title for this kind of card?